site2/docs/administration-proxy.md

```diff
@@ -55,7 +55,9 @@ To start the proxy:
 
 ```bash
 $ cd /path/to/pulsar/directory
-$ bin/pulsar proxy
+$ bin/pulsar proxy \
```

Comment:
Is it necessary to update here?

Reply:
I think the it forgot add start command previously.